curator-x-discovery-server/src/test/java/org/apache/curator/x/discovery/server/jetty_resteasy/TestStringsWithRestEasy.java

```diff
@@ -65,7 +65,11 @@ public void         setup() throws Exception
 
         port = InstanceSpec.getRandomPort();
         server = new Server(port);
-        Context root = new Context(server, "/", Context.SESSIONS);
+        ServletContextHandler root = new ServletContextHandler(ServletContextHandler.SESSIONS);
+        root.setContextPath("/");
+        server.setHandler(root);
```

Comment:
nit:
Would be cleaner to set up the root object completely, and call setHandler after.